{"name": "test_invalid_login", "status": "failed", "statusDetails": {"message": "AssertionError: Expected error message not shown\nassert '' != ''", "trace": "driver = <appium.webdriver.webdriver.WebDriver (session=\"c80443c2ad2f612c858b150a972d9c2f02e54120\")>\n\n    def test_invalid_login(driver):\n        lp = LoginPage(driver)\n        # locked_out_user always fails\n        lp.login(\"locked_out_user\", \"secret_sauce\")\n        err = lp.get_error()\n>       assert err != \"\", \"Expected error message not shown\"\nE       AssertionError: Expected error message not shown\nE       assert '' != ''\n\ntests\\test_login.py:18: AssertionError"}, "attachments": [{"name": "screenshot-android", "source": "47a9cad6-15e1-4ff4-8db5-5d8123904262-attachment.png", "type": "image/png"}], "start": 1757837437915, "stop": 1757837462499, "uuid": "0e32caa8-a484-43a8-b2e1-5ad2a3877d06", "historyId": "6485cd1aa206d185bd707f422a9a9238", "testCaseId": "6485cd1aa206d185bd707f422a9a9238", "fullName": "tests.test_login#test_invalid_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "DB-L-174"}, {"name": "thread", "value": "2256-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}], "titlePath": ["tests", "test_login.py"]}